### DISCUSSION TITLE
Templater & Moment.js

### DISCUSSION THREAD

**Discussion by Gnossos** [SilentVoid13/Templater#1254] [Help]
Two Questions:

1. Exactly how does Templater provide access to moment.js? Is anything more than simply including the moment.js libraries and making moment.js calls legal within <% ... %> blocks?
2. Now that moment.js has been deprecated, are there any plans to switch (or just support) to a different time & date interface?

**Comment by Zachatoo** [comment]
1. `moment` is a global provided by Obsidian, so Templater can access it for free. You can also access other globals like `app`.
1. There's no way to remove access to the global `moment` without getting hacky by removing and re-adding it, which sounds like a bad idea. There are no plans to add support for another similar library. If Obsidian switches, so will Templater by nature.
